### ISSUE TITLE
Mentors on the Rust track

### ISSUE THREAD

**Issue by hallodaniel** [exercism/exercism#4776]
Hi,

I have been waiting for some time now for a mentor to review my solutions to the first two exercises on the Rust track but nothing has happened so far. 

Are the mentors totally overloaded (32 mentors for ~11k students does not seem a lot), should I be a bit more patient or should I just move on with other exercises?

Daniel

**Comment by NobbZ** [comment]
What do you mean by "some time"?

I had a submission reviewed after 24h.

But what do you mean you have submitted the first 2? Usually you submit a single core exercise for review at a time.

If though you mean you have submitted side-exercises, then getting review for them is a very low priority task. Core submissions are more important as they block students progress.

**Comment by hallodaniel** [comment]
> What do you mean by "some time"?

Around two weeks

> I had a submission reviewed after 24h.
> 
> But what do you mean you have submitted the first 2? Usually you submit a single core exercise for review at a time.

Bracket Push and Pythagorean Triplet 

> If though you mean you have submitted side-exercises, then getting review for them is a very low priority task. Core submissions are more important as they block students progress.

Yes thank you

**Comment by NobbZ** [comment]
So you have 2 core exercises with pending review?

Also I'm wondering why you are saying they were your first submissions. Both exercises are somewhere after the first couple of exercises. `pythagorean-triplet` is the fourth if I haven't counted wrong. `bracket-push` is a little bit later, with `saddle-points` inbetween.

Are you sure you are in mentored mode? If so, have you already completed `hello-world`? And can you please share the UUIDs of both submissions that have a pending review? Or if you are not sure how to figure out the UUID, just give a link to your submissions.

**Comment by hallodaniel** [comment]
> So you have 2 core exercises with pending review?

The site says they are "in progress" and that the next step is for a mentor to leave feedback

> Also I'm wondering why you are saying they were your first submissions. Both exercises are somewhere after the first couple of exercises. `pythagorean-triplet` is the fourth if I haven't counted wrong. `bracket-push` is a little bit later, with `saddle-points` inbetween.

The order I see is 1\hello world 2\reverse string 3\bracket push 4\ p triplet. Saddle points is 6.

> Are you sure you are in mentored mode?

I guess so yes, I could switch to independent mode and exercises after 4\ p triplet are locked

> If so, have you already completed `hello-world`? 

Yes

>And can you please share the UUIDs of both submissions that have a pending review? Or if you are not sure how to figure out the UUID, just give a link to your submissions.

0357cc64b74b4b4295b3a270bfddf6b3
7413bee1c87149849e6ce52a6746f886
Do those work?

**Comment by NobbZ** [comment]
Those look correct, I'll point the rust mentors to this issues as soon as I have slack available.

**Comment by hallodaniel** [comment]
> Those look correct, I'll point the rust mentors to this issues as soon as I have slack available.

Awesome thank you!

**Comment by F3PiX** [comment]
@NobbZ  Rust is currently making changes to the order of the cores in the [Track Anatomy](https://exercism.io/blog/track-anatomy-project). As a result, sometimes there's more than one core exercise in progress. And for the student, the newly added core will look like "the next unlocked core exercise".

**Comment by NobbZ** [comment]
Thanks for the info @F3PiX, I know about the changes in progress, but I forgot about how the website behaves in that cases.

**Comment by F3PiX** [comment]
I can imagine that that slips one's mind :-)

 I forgot to add that the descripted order is all expected behaviour with regard to the changes made.

**Comment by NobbZ** [comment]
Hello @hallodaniel!

I've talked with a mentor, he said, there are only 3 active mentors currently and quite a big backlog.

The only reason that I received feedback in a day, was because I was lucky. I did an exercise, that was moved around recently and the mentor wanted to check if its easily reviewable.

I think we'll have both to wait for further review, as my recent submission is not `reverse-string` anymore, but later in the track.
